xlators/debug/io-stats/src/io-stats.c

```diff
@@ -241,6 +242,7 @@ is_fop_latency_started(call_frame_t *frame)
         conf = this->private;                                                  \
         if (conf && conf->measure_latency) {                                   \
             timespec_now(&frame->begin);                                       \
+            conf->last_fop_hit = gf_time();                                    \
```

Comment:
Can't we copy the value from frame->begin?

Reply:
To use frame->begin we have to change it in seconds because during io_stats_dump we do call gf_time to measure the interval that's why we need to call gf_time during start.